src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -382,6 +394,12 @@ def makeTapeSubscriptions(self, workflow):
         :param workflow: a MSOutputTemplate object representing a workflow
         :return: the MSOutputTemplate object itself (with the necessary updates in place)
         """
+        # Do not make relval workflow subscription if not enabled
+        if workflow['IsRelVal'] and self.msConfig['enableRelValDisk']:
+            self.logger.info("Skipping tape subscription for relval workflow",
+                             workflow['RequestName'])
+            return workflow
```

Comment:
Similarly to the comment above, I fear that if we simply return here, we might miss the execution of some logic in this function of the pipeline.
Perhaps another way to implement it more inline with the current implementation of this microservice would be to check for relval disk data placement right at the beginning of this for loop:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/MicroService/MSOutput/MSOutput.py#L394
`continue`ing to the next loop iteration if necessary.